global.json

```diff
@@ -5,12 +5,11 @@
   "tools": {
     "dotnet": "7.0.100-alpha.1.21558.2",
     "runtimes": {
-      "dotnet/x64": [
-        "2.1.30",
+      "dotnet": [
         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
       ],
-      "dotnet/x86": [
-        "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
+      "dotnet/x64": [
+        "2.1.30"
```

Comment:
I'm not positive but suspect we have scripts etc. (like activate.*) that expect .dotnet/x86 for some of our special-case testing. Nothing should be looking for .dotnet/x64 and I think that's where 2.1.30 will be installed w/ this setup.

Might be worth trying

``` json
"runtimes": {
  "dotnet": [
    "2.1.30",
    "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
   ],
  "dotnet/x86": [
    "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
  ],
  "aspnetcore": [
    "3.1.21"
  ]
}
```

This suggestion should install all shared framework bits for the current system (Arm[64] / x64 or whatever) and add the latest Microsoft.NETCore.App for x86. But, it may not work on Arm64 machines because x86 emulation isn't available (right ❔). We don't have a way to say "please add x86 bits if this is an x64 machine" ☹️

But, maybe we can stop testing x86 on build agents. @Tratcher❔

Reply:
I think I completely misunderstood what this part of global.json does. Are there some docs for what this `runtimes` section is used for?

Reply:
https://github.com/dotnet/arcade/blob/56bac3e40e405eacba76d048aaff104c01f4d8ee/Documentation/ArcadeSdk.md#globaljson covers some of it.

But, I checked deeper and the infrastructure we use from dotnet/arcade
1. Ignores `dotnet/x86` entries unless on Windows
2. Does **not** ignore `dotnet/x86` entries on Windows Arm64 (☹️ but building there is less of a priority)
3. Doesn't seem to change the installation location for `dotnet/x64` entries on Windows
4. Adds the specified architecture (the bit after the slash) to the installation location on Linux. But, I very much doubt we attempt to do x64 testing on Arm64 build agents or in Arm64 local builds.
5. Defaults to using the current `MSBuild` `$(Platform)` (ignoring `AnyCpu`) when no architecture portion (the bit after the slash) is specified. This may be a problem for building on Arm64.

Therefore, my suggestion might be fine because it maintains `dotnet/x86` and avoids per-x64 downloads on Arm64 build agents. Might not need the `dotnet/x86` bit if @Tratcher thinks we can eliminate x86 testing on Windows.

But, please confirm a local build on Arm64 downloads the Arm64 version of Microsoft.NETCore.App. We may need to get some help here from the Arcade team.

Reply:
x64 is far more common than x86, and I'm not are of any x86 specific issues we've had. I would hope there's still at least one x86 test run somewhere, even if its on CTI.

Reply:
@omajid please try my suggestion because we need to know if leaving the `dotnet/x86` entry here breaks the Linux Arm64 build. If it doesn't, that change is easy and preserves the Server Comparison tests on Windows.

Reply:
@dougbu Your suggestions for global.json works for me!

Reply:
Please update your branch and push to your fork. Then, we can confirm the suggestion doesn't break other bits 😃

Separately, are your local builds looking solid enough that your next iteration should be ready for review❔